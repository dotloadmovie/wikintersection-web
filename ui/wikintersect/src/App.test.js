import React from 'react';
import ReactDOM from 'react-dom';
import AppConnected from './App';

import { shallow } from 'enzyme';
import {Provider} from "react-redux";
import store from "./stores/Store";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Provider store={store}>
      <AppConnected />
    </Provider>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('displays correctly', () => {
  const component = shallow(<Provider store={store}>
    <AppConnected />
  </Provider>)

  expect(component).toMatchSnapshot();
})
