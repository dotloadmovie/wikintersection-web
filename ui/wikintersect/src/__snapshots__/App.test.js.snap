// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays correctly 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
      "storeState": Object {
        "compareResult": null,
        "currView": "search",
        "search0": Object {
          "results": null,
          "rowKey": null,
          "search": null,
          "serverRequestInFlight": false,
          "value": null,
        },
        "search1": Object {
          "results": null,
          "rowKey": null,
          "search": null,
          "serverRequestInFlight": false,
          "value": null,
        },
        "serverRequestInFlight": false,
      },
    }
  }
>
  <Connect(App) />
</ContextProvider>
`;
